### DISCUSSION TITLE
How to add popup for the drawing marker tool in the left menu?

### DISCUSSION THREAD

**Discussion by zxdawn** [opengeos/leafmap#556] [Q&A] [answered]
It's nice to see the html exported by leafmap has a nice left tool menu. Is it possible to click the marker to show the latitude and longitude?

<img width="1076" alt="image" src="https://github.com/opengeos/leafmap/assets/30388627/b4636dda-00c4-47c9-a46e-13d5d1fe0f86">

**Comment by giswqs** [comment, marked as answer]
See https://leafmap.org/notebooks/34_add_points_from_xy/
![image](https://github.com/opengeos/leafmap/assets/5016453/d8f39556-be8b-4c31-be8e-072a2c316778)
